Grenache.Utils/RpcActionHandler.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System.Reflection;
+using Newtonsoft.Json.Linq;
+
+namespace Grenache.Utils;
+
+public class RpcActionHandler(Assembly targetAssembly)
+{
+    public Func<object, object> HandleAction(string json)
+    {
+        var jsonObject = JObject.Parse(json);
+        
+        if (!jsonObject.TryGetValue("action", StringComparison.OrdinalIgnoreCase, out JToken actionToken))
+        {
+            throw new ArgumentException("JSON must include an 'action' property.");
+        }
+        var action = actionToken.ToString();
+
+        if (!jsonObject.TryGetValue("args", StringComparison.OrdinalIgnoreCase, out JToken argsToken))
+        {
+            throw new ArgumentException("JSON must include an 'args' property.");
+        }
+        var args = (JArray)argsToken;
+
+        var theMethod = FindMethodInAssembly(action, args);
+
+        if (theMethod == null)
+        {
+            throw new InvalidOperationException($"No suitable method found for action: {action}");
```

Comment:
in bfx we have ERR_... format for messages, so more like api codes, can we do the same here please? e.g. ERR_ACTION_NOT_FOUND

Reply:
done